### DISCUSSION TITLE
en/docs/getting-started/installation/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#95] [General]
# en/docs/getting-started/installation/

We provide a starter theme template for you to create a HB site quickly.

https://hb.hugomods.com/en/docs/getting-started/installation/

**Comment by FuadEfendi** [comment]
Hi Razon, I have error when I try to start server: "content/blog/tutorial/index.md:69:1 failed to extract shortcode: template for shortcode "bootstrap/alert" not found"

**Reply by razonyang** [reply]
Hi, I just follow the steps to create a new site on another OS (Windows), but didn't reproduce any errors, could you please provide details about setting up the site? 
It would be better if you could provide a public repo for reproducing this.

Another clue is that the shortcode comes from [module](https://github.com/hugomods/bootstrap), it seems some modules is missing from your environment. `hugo mod clean` and then `hugo mod get -u` may work, I'm not sure.

**Reply by FuadEfendi** [reply]
Sorry my fault... and thank you Razon for instant help!

I had to upgrade Hugo from 0.104.0+extended to 0.111.3+extended, it works now! I am creating my personal site based on this modular theme (and it is really hard to find theme using primarily GO modules!): http://www.FuadEfendi.com
- I'll be adding educational content there, Java / Microservices / NoSQL

I am very happy with previous theme, I know you implemented Algolia integration which I have not tried yet.

**Reply by razonyang** [reply]
Np :)
